### PR TITLE
Changed the Kalman Filter tutorial.

### PR DESCRIPTION
**Pull request by jmbarr**
I moved stuff around and added/removed to emphasise the introduction to Stone Soup, rather than a tutorial on the KF (though it can still be used as such). A brief check seems to suggest that some of the markdown doesn't make it into the Notebook... suggestions welcome.

### CODE REVIEW

File: docs/tutorials/01_KalmanFilterTutorial.py [hunk 1]
```diff
@@ -2,298 +2,195 @@
...
-#
-# .. math::
-#           p(x) &= \int_{X} p(x|x')p(x')dx'\\
-#           P(X|Y) &= \frac{P(Y|X)P(X)}{P(Y)}
+# Update
+# ^^^^^^
 #
-# We get:
+# We assume a sensor measurement is generated by some stochastic process represented by a function,
+# :math:`\mathbf{z}_k = h(\mathbf{x}_k, \boldsymbol{\nu}_k)` where :math:`\boldsymbol{\nu}_k` is the noise.
```
Comment (oharrald-Dstl) [inline docs/tutorials/01_KalmanFilterTutorial.py:55]: '\boldsymbol' requires a LaTeX package. Text appears in red on generated html page.

File: docs/tutorials/01_KalmanFilterTutorial.py [hunk 2]
```diff
@@ -2,298 +2,195 @@
...
+#                     0 & 1\\
+#                \end{bmatrix} \omega
 #
-# Next we'll create our model which will describe errors of measuring in the observable state, and
-# how the hidden state maps to observable state. In this case we can simply map the :math:`x` and
-# :math:`y` directly.
-from stonesoup.types.detection import Detection
-from stonesoup.models.measurement.linear import LinearGaussian
+# where :math:`\omega` is set to 0.75 initially (but again, feel free to play around).
 measurement_model = LinearGaussian(
```
Comment (oharrald-Dstl) [inline docs/tutorials/01_KalmanFilterTutorial.py:194]: (something I probably should have done) could add kwargs here to make it more clear.

File: docs/tutorials/01_KalmanFilterTutorial.py [hunk 3]
```diff
@@ -409,8 +309,4 @@
     ax.add_artist(ellipse)
 fig
 
-# sphinx_gallery_thumbnail_number = 4
-
-# %%
-# There are situations in which linearisation of the problem is not useful/possible.
-# For this, we require an adjusted process, as tackled by the **Extended Kalman Filter**.
+# sphinx_gallery_thumbnail_number = 4
```
Comment (oharrald-Dstl) [inline docs/tutorials/01_KalmanFilterTutorial.py:312]: Thumbnail doesnt appear on Tutorials page.

File: docs/tutorials/01_KalmanFilterTutorial.py (hunk 2 above)
Comment (jmbarr) [reply docs/tutorials/01_KalmanFilterTutorial.py:194]: Not sure what you mean, but go ahead

File: docs/tutorials/01_KalmanFilterTutorial.py (hunk 3 above)
Comment (jmbarr) [reply docs/tutorials/01_KalmanFilterTutorial.py:312]: I haven't taken the trouble to work out what this means - again, happy for you to go ahead and make it work.

File: docs/tutorials/01_KalmanFilterTutorial.py (hunk 1 above)
Comment (jmbarr) [reply docs/tutorials/01_KalmanFilterTutorial.py:55]: Works ok for me. I guess not everyone gets amsfonts? Anyone else?

Could change symbol, leave unbold, make Roman...

Comment (sdhiscocks) [reply docs/tutorials/01_KalmanFilterTutorial.py:55]: Looks okay on my system as well. I thought they were rendered by a javascript library, so shouldn't matter what packages you have locally.

Comment (oharrald-Dstl) [reply docs/tutorials/01_KalmanFilterTutorial.py:55]: Weird, looks fine on mine now. Please ignore my comment

**Pull request merged**